### ISSUE TITLE
Add checkbox for livestreaming

### ISSUE THREAD

**Issue by netravnen** [bornhack/bornhack-website#620]
On the event page. I recommend adding a boolean checkbox for indicating if the event is live streamed or not.

My reasoning being for especially the 2020 event. Where the speakers tent is limited to ~60 chairs. And not all events will be recorded for later viewing.
It makes sense, IMHO, to improve the communication degree. Example, the Jepordy rounds are live streamed. But this is not 100 percent clear when viewing the program entry [https://bornhack.dk/bornhack-2020/program/hacker-jeopardy-round-1/] that the rounds will be live streamed. Apart from that it/they WILL NOT be recorded for later viewing on e.g. YouTube Bornhack channel.

**In short**

Add a checkbox above or below the current meta information showing if a program entry (presentation) will be live streamed or not. 

![Screenshot_20200812-202116_Chrome.jpg](https://user-images.githubusercontent.com/1938389/90052428-680e4100-dcd9-11ea-923d-ff5f54742e49.jpg)
